product_supplierinfo_for_customer_sale/README.rst

```diff
@@ -35,30 +35,30 @@ To use this module, you need:
 - Go to product and configure *Partner product name* and *Partner product code*
   for each selected customer.
 
-.. figure:: https://raw.githubusercontent.com/OCA/sale-workflow/11.0/product_supplierinfo_for_customer_sale/static/description/configuration_customer.png
+.. figure:: static/description/configuration_customer.png
```

Comment:
It can be that you have an old README generator tool? This shouldn't be modified

Reply:
And any way, why regenerating it?

Reply:
This is still wrong